src/ray/gcs/gcs_server/test/gcs_actor_scheduler_test.cc

```diff
@@ -30,6 +30,8 @@ namespace gcs {
 class GcsActorSchedulerTest : public ::testing::Test {
  public:
   void SetUp() override {
+    RayConfig::instance().initialize(
+        "{\"scheduler_top_k_absolute\": 1, \"scheduler_top_k_fraction\": 0}");
```

Comment:
Q: Does it make sense to set this here though? Does it mean we are not unit-testing the code correctly? (since by default k is 10% of nodes, we are testing the different logic all the time)? 

My understanding is if we mock GetBestNode correctly, this value shouldn't matter.